orchestra/__init__.py

```diff
@@ -17,7 +17,7 @@ def write(self, message):
 def _main(argv):
     args = main_parser.parse_args(argv)
 
-    logger.remove(0)
+    logger.remove(None)
```

Comment:
`None` is [default](https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.remove)